Assignment2/xinying-huang-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": {
+    "base": "http://www.oeg-upm.net/Organization",
+    "vocab": "http://www.oeg-upm.net/Organization#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    
+    "vocab:atTime": {
+      "@type": "xsd:dateTime"
+    },
+    "vocab:hasTemperature": {
+      "@type": "xsd:integer"
+    },
+    "vocab:hasName": {
+      "@type": "xsd:string"
+    }
+  },
+  "Class01": {
+    "@id" : "http://www.oeg-upm.net/Class01",
+    "vocab:includes": {
+      "Sensor029":{
```

Comment:
You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you to remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes. However, when you have to specify a relationship between resources is when you have to define it as an attribute, for example as you did with the attribute "hasMeasurement" for defining the relationship between Sensor029 and Measurment8401. In the case of Sensor and Computer they might be defined into an array.
This comment also applies for "Measurement8401", "Computer101", and "User10A".